classification/classifier_main.py

```diff
@@ -40,3 +45,7 @@ def is_page_stampifiable(self):
         ''' returns the is_stampifiable flag'''
         self.classify()
         return self.is_stampifiable
+
+    def get_website_type(self):
+        tokenized_words = word_tokenize(self.webpage_title)
+        return any(token.isnumeric() for token in tokenized_words)
```

Comment:
Does it handle cases like "one", "two" etc. as well?

Reply:
It doesn't ! This is a very basic version - will have to improve on this with other metrics .

Reply:
Sounds good! Just leave a comment explaining the cases it handles.

Reply:
updated accordingly!